ext/webextension/src/browser_action/main_popup.js

```diff
@@ -509,4 +509,40 @@ document.querySelector('#siteconfig_show').addEventListener('click', function()
     ui.hide('#siteconfig_show');
 });
 
+function indetifierUpdate()
+{
+    var text = document.querySelector("#username").value;
+    text += document.querySelector('#masterkey').value;
+    var identifier = document.querySelector('#identifier');
+    if (text == '' || text == null || text.length < 6)
+    {
+        identifier.innerHTML = '';
+    }
+    else if (text.length >=6)
+    {
+        var emojiStr = "&#1285";
```

Comment:
I think these /var/'s should be /let/  
they are written as if being block scoped (and probably sold be) but /var/ will hoist them to the function scope

Reply:
I have updated the code as requested.